multicluster/controllers/multicluster/commonarea/remote_common_area_test.go

```diff
@@ -49,6 +111,12 @@ func TestMemberAnnounce(t *testing.T) {
 		Namespace:          "cluster-a-ns",
 		connected:          false,
 		localClusterClient: nil, // Not used for this test
+		leaderStatus: mcsv1alpha1.ClusterCondition{
+			Message: "Member Connected",
+			Reason:  "Disconnected",
+			Status:  v1.ConditionFalse,
+			Type:    mcsv1alpha1.ClusterReady,
```

Comment:
The default leader status's type is `multiclusterv1alpha1.ClusterIsLeader`.
```suggestion
			Type:    mcsv1alpha1.ClusterIsLeader,
```

Please check the default value here, it's confused to see `Member Connected` and `Disconnected`.
```
	remote.leaderStatus.Type = multiclusterv1alpha1.ClusterIsLeader
	remote.leaderStatus.Status = v1.ConditionFalse
	remote.leaderStatus.Message = "Leader cluster added"
```